code/Modules/Items/PlayerInventory.cs

```diff
@@ -64,5 +70,20 @@ public void Add(Item item, int position = 0)
             base.Add(item, position);
             StartHolding(HoldingSlot);
         }
+
+        /// <summary>
+        /// Updates the view model's hold type to match the holding item
+        /// </summary>
+        private void RefreshViewModel()
+        {
+            if (null != HoldingItem)
+            {
+                ViewModel.SetHoldType(HoldingItem.HoldType);
+            }
+            else if (null == HoldingItem)
+            {
+                ViewModel.SetHoldType(HoldType.None);
+            }
```

Comment:
HolfingItem == null ? blah : blouh
Better
ViewModel.SetHoldType(HoldingItem == nul ? blah : blouh)
Or you can set the hold type before